convert_state.py

```diff
@@ -34,7 +34,8 @@ def main():
             print('try again')
 
 
-def convert_state_to_abbreviation(dictionary ):
+def convert_state_to_abbreviation(dictionary):
+    """Function to get state name and give abbreviation"""
     user_input = input('Enter state name: ').  capitalize()
```

Comment:
there is space between the . and the capitalize()

Reply:
fixed.